### DISCUSSION TITLE
Migrations

### DISCUSSION THREAD

**Discussion by prabhatsharma** [prest/prest#537] [API Server] [answered]
How does migration work? I could not find documentation on it. Happy to contribute and create it, if something does not exist and someone can explain be a bit.

**Comment by tiaguinho** [comment]
Hi @prabhatsharma.
The docs about migrations are inside configurations section (https://docs.prestd.com/configurations/#migrations)

**Comment by avelino** [comment]
@prabhatsharma were you able to use it?

**Comment by prabhatsharma** [comment, marked as answer]
Thanks a lot @tiaguinho  @avelino . I did see this URL but could not understand how to use it. Currently, I use a .sql file with all the CREATE statements. How do I tell prest on how to do the migration. What should be contents of the folder?

**Reply by avelino** [reply]
@prabhatsharma great point, I feel ashamed to think that everyone could use it without a simple documentation - for me it is natural because I use it every day, sorry.

I wrote a page in our documentation explaining how to use the prest migration system.

Your feedback on the documentation is welcome

https://docs.prestd.com/migrations/
